src/plugins/xerces/README.md

```diff
@@ -16,36 +16,36 @@ formatted files. It uses a general format which:
 - Maps key names to XML elements
 - Maps key values to textual content of XML elements
 - Maps metakeys to XML attributes. Metakey name = attribute name, Metakey value
- 	= attribute value
+  = attribute value
 - Ignores XML comments
 
 ## Usage
 
 To mount an XML file we use:
 
-	kdb mount file.xml user/test/file xerces
+    kdb mount file.xml user/test/file xerces
 
 The strength and usage of this plugin is that it supports arbitrary XML files and
 does not require a specific format. Given the following example of an XML file:
 
-	<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
-	<xerces>foo
-	  <bar meta="da_ta">bar</bar>
-	</xerces>
+    <?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
Shouldn't we use the XML syntax highlighting here?

Reply:
That makes sense. I specified the language of the code section in commit 2a409be.